src/main/java/domain/Length.java

```diff
@@ -0,0 +1,24 @@
+package domain;
+
+import exception.NegativeLengthException;
+
+public class Length {
+    private final int length;
+    private static final int LENGTH_MIN_BOUNDARY = 0;
```

Comment:
```suggestion
    private static final int LENGTH_MIN_BOUNDARY = 0;
    private final int length;
```
변수의 선언 순서는 상수 - 인스턴스 변수 순으로 선언합니다.